wcfsetup/install/files/lib/system/mail/Mail.class.php

```diff
@@ -497,8 +396,19 @@ public function setContentType($contentType) {
 	 * @param	string		$header
 	 */
 	public function setHeader($header) {
```

Comment:
The new implementation is bogus. `setHeader` expected to receive a string that possibly contains multiple headers. You would need to parse this string according to RFC 5322 to extract the headers.